### PR TITLE
Create airship.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sources/airship.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+description: Adding Airship’s unique information to Amplitude gives you a full view of what’s going on in and around your app.
+---
+
+Airship Real-Time Data Streaming (RTDS) delivers user-level information about push sends, conversions, and uninstall events. Adding Airship’s unique information to Amplitude gives you a full view of what’s going on in and around your app.
+
+!!!note "Other Amplitude + Airship integrations"
+
+    This integration imports Airship data into Amplitude. Amplitude offers one other integrations with AppsFlyer: 
+
+    - [Send cohorts to AppsFlyer](/data/destinations/airship-cohort)
```
Comment (ampcynthli) [inline docs/data/sources/airship.md:12]:
```suggestion
    This integration imports Airship data into Amplitude. Amplitude offers one other integrations with Airship: 

    - [Send cohorts to Airship](/data/destinations/airship-cohort)
```

File: docs/data/sources/airship.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## About Airship
+
+[Airship](https://www.airship.com/) provides mobile advertising attribution and analytics, helping marketers to pinpoint their targeting optimize their ad spend and boost their ROI. Send your AppsFlyer data to Amplitude to see how your different acquisition sources are performing.
```
Comment (ampcynthli) [inline docs/data/sources/airship.md:16]: Is this accurate to Airship or is it AppsFlyer's description?

**Review by ampcynthli — CHANGES_REQUESTED** [review]

File: docs/data/sources/airship.md (hunk 2 above)
Comment (Brandoncharleskhoo1) [reply docs/data/sources/airship.md]: Whoops! Good catch Cynthia. I updated the blurb changes.

File: docs/data/sources/airship.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+!!!note "Other Amplitude + Airship integrations"
+
+    This integration imports Airship data into Amplitude. Amplitude offers one other integrations with Airship: 
+
+    - [Send cohorts to Airship](/data/destinations/airship-cohort)
+
+## About Airship
+
+[Airship](https://www.airship.com/) The Airship App Experience Platform’s no-code approach and intuitive visual UIs empower the entire team to create and adapt customer journeys, native app experiences and app UX to capture more value from mobile apps.
```
Comment (ampcynthli) [inline docs/data/sources/airship.md:16]:
```suggestion
The [Airship](https://www.airship.com/) App Experience Platform’s no-code approach and intuitive visual UIs empower the entire team to create and adapt customer journeys, native app experiences and app UX to capture more value from mobile apps.
```
Fixed hyperlink to integrate into descriptor

**Review by ampcynthli — APPROVED** [review]
just a quick change, lgtm

**Review by Leoaqr — APPROVED** [review]

**Comment by bgiori** [comment]
@Brandoncharleskhoo1 Is this still in the works?

**Comment by Brandoncharleskhoo1** [comment]
Hi @nirmal91 @bgiori or @justin-fiedler , can you please review?

**Review by justin-fiedler — APPROVED** [review]

**Comment by kevinpagtakhan** [comment]
@Brandoncharleskhoo1 do we still need this PR? Looks like we're ready to merge?

**Pull request closed without merging**